lib/external/io.dart

```diff
@@ -23,11 +25,43 @@ ASTNode funcFree(Runtime runtime, ASTNode self, List<ASTNode> args) {
   Scope scope = getScope(runtime, self);
 
   args.forEach((arg) {
-    if (arg is! VariableNode)
-      return;
+    if (arg is! VariableNode) return;
 
-    scope.variableDefinitions.removeWhere((varDef) => varDef.variableName == arg.variableName);
+    scope.variableDefinitions
+        .removeWhere((varDef) => varDef.variableName == arg.variableName);
   });
 
   return INITIALIZED_NOOP;
-}
\ No newline at end of file
+}
+
+StringNode funcCwd(Runtime runtime, ASTNode self, List<ASTNode> args) {
+  StringNode res = StringNode();
+  Directory current = Directory.current;
```

Comment:
This line is redundant. If current is only used once, i wouldn't say its worth making a separate variable for it. Use Directory.current directly